qemu/tests/cfg/uefishell.cfg

```diff
@@ -51,6 +51,46 @@
                     command_ping6_args = "form_ping6_args()"
                     check_result_show6 = "(\w+::\w+:\w+:\w+:\w+)/64"
                     check_result_ping6 = "0.*packet loss"
+        - check_net_boot_entry:
+            bootindex_nic1 = 1
+            Windows:
+                cdroms = ""
+            test_scenarios = "connect bcfg"
+            command_connect = "connect -r"
+            command_bcfg = "bcfg boot dump"
+            check_result_bcfg = "UEFI\s+PXEv4, UEFI\s+PXEv6, UEFI\s+HTTPv4, UEFI\s+HTTPv6"
+            variants:
+                - with_one_serial:
+                    serials = "vs1"
+                    serial_type_vs1 = isa-serial
+                - with_two_serials:
+                    serials = "vs1 vs2"
+                    serial_type_vs1 = isa-serial
+                    serial_type_vs1 = isa-serial
```

Comment:
I guessed this should be serial_type_vs2
Or  you could set serial_type = isa-serial under line62 as a global setting

Reply:
yes, changed to serial_type_vs2, many thanks.